aesd-char-driver/main.c

```diff
@@ -197,12 +198,106 @@ ssize_t aesd_write(struct file *filp, const char __user *buf, size_t count, loff
     return ret;
     
 }
+
+loff_t aesd_llseek( struct file *filp, loff_t offset, int whence )
+{
+    loff_t status;
+    struct aesd_dev *dev = filp->private_data;
+    
+    if (mutex_lock_interruptible(&aesd_device.mtx_lock)) {
+        status = -ERESTARTSYS;
+        return status;
+    }
+    status = fixed_size_llseek(filp, offset, whence, dev->c_buff.total_buff_size);
```

Comment:
@akshithaluguri22 what is a way you could avoid mutex locking on the call to `fixed_size_llseek`?